.github/workflows/release.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 env:
   PLUGIN_NAME: FinSearch
-  PLUGIN_MIN_REQUIREMENT: ^6.4.6
+  PLUGIN_MIN_REQUIREMENT: ~6.4.6
```

Comment:
We do not want to support `6.5` without manually defining it, as there will probably be breaking changes.